config/localTemplate.go

```diff
@@ -247,6 +247,10 @@ type Local struct {
 	// telemetry ( when enabled ). Defined in seconds.
 	PeerConnectionsUpdateInterval int `version[5]:"3600"`
 
+	// HeartbeatUpdateInterval defines the interval at which the heartbeat information is being sent to the
+	// telemetry ( when enabled ). Defined in seconds.
+	HeartbeatUpdateInterval int `version[26]:"600"`
```

Comment:
I know we use integers for all of these, but I'm curious... is there any reason not to use `time.Duration` so that you can use values like `10m`?

Reply:
Seems strange that we use time.Duration, but only use the nanos input for the value.

Reply:
We also have `LogArchiveMaxAge string` which uses `time.ParseDuration` :)

Reply:
Oh, and we have settings that take in a number of seconds using int, int64 and uint as the storage type

Reply:
You're right, it doesn't work: https://go.dev/play/p/BMc1fLrzAMe